packages/design-system/src/styleguide/colors.stories.mdx

```diff
@@ -91,6 +91,32 @@ import ColorCircles from './ColorCircles.vue';
 	</Story>
 </Canvas>
 
+## Neutral
+
+<Canvas>
+	<Story name="neutral">
+		{{
+			template: `<color-circles :colors="['--color-neutral']" />`,
+			components: {
+				ColorCircles,
+			},
+		}}
+	</Story>
+</Canvas>
+
+## White
+
+<Canvas>
+	<Story name="white">
+		{{
+			template: `<color-circles :colors="['--color-white']" />`,
```

Comment:
unless there's a usecase I don't understand here, would rather we stick to foreground and background xlight, so we could continue to separate between those usecases... Plan is for figma designs to follow this..

Reply:
Thank you for letting me know. Removed. Updated to background xlight.